django_app/view_a_suspected_breach/views.py

```diff
@@ -1,10 +1,43 @@
+from typing import Any
+
 from core.sites import require_view_a_breach
-from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.models import User
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
+from django.shortcuts import reverse
 from django.utils.decorators import method_decorator
 from django.views.generic import TemplateView
 
+from .mixins import ActiveUserRequiredMixin, StaffUserOnlyMixin
+
 
-@method_decorator(login_required, name="dispatch")
 @method_decorator(require_view_a_breach(), name="dispatch")
-class ViewABreachView(TemplateView):
+class ViewABreachView(LoginRequiredMixin, ActiveUserRequiredMixin, TemplateView):
     template_name = "view_a_suspected_breach/landing.html"
+
+
+@method_decorator(require_view_a_breach(), name="dispatch")
+class AdminViewABreachView(LoginRequiredMixin, StaffUserOnlyMixin, TemplateView):
+    template_name = "view_a_suspected_breach/user_admin.html"
+
+    def get_context_data(self, **kwargs: object) -> dict[str, Any]:
+        context = super().get_context_data(**kwargs)
+        context["pending_users"] = User.objects.filter(is_active=False, is_staff=False)
+        context["accepted_users"] = User.objects.filter(is_active=True)
+        return context
+
+    def get(self, request: HttpRequest, **kwargs: object) -> HttpResponse:
+        user_objects = User.objects.all()
+
+        if update_user := self.request.GET.get("accept_user", None):
+            user_to_accept = user_objects.get(id=update_user)
```

Comment:
Can just do the isolated query down here

```suggestion
            user_to_accept =User.objects.get(id=update_user)
```